packages/terra-form-select/src/_Menu.jsx

```diff
@@ -163,14 +207,35 @@ class Menu extends React.Component {
         return;
       }
 
-      if (hasNoResults && searchValue) {
```

Comment:
I take it that if searchValue is null then we assume that hasNoResults is false?

Reply:
Yeah, I pulled this code change from this PR: https://github.com/cerner/terra-core/pull/2217/files

Reply:
Forgot I wrote that. Yeah, I think this line of code is valid.